spec/system/support_interface/staff_spec.rb

```diff
@@ -27,11 +28,29 @@
     then_i_sign_out
 
     when_i_visit_the_invitation_email
-    and_i_fill_password
-    and_i_set_password
+    then_i_am_taken_to_the_login_page
+  end
+
+  it "allows inviting a user with azure ad deactivated" do
+    when_azure_ad_authentication_is_not_active
+    given_i_am_authorized_as_a_support_user
+    when_i_visit_the_staff_page
     then_i_see_the_staff_index
 
-    then_i_see_the_accepted_staff_user
+    when_i_click_on_invite
+    then_i_see_the_staff_invitation_form
+
+    when_i_fill_email_address
+    and_i_fill_name
+    and_i_choose_support_console_permission
+    and_i_send_invitation
+    then_i_see_an_invitation_email
+    then_i_see_the_staff_index
+    then_i_see_the_invited_staff_user
+    then_i_sign_out
+
+    when_i_visit_the_invitation_email
+    then_i_am_taken_to_the_login_page
```

Comment:
```suggestion
    and_i_fill_password
    and_i_set_password
    then_i_see_the_staff_index

    then_i_see_the_accepted_staff_user
```

It looks like this is missing from this test ([taken from the one we've got currently](https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/a17c21a393333c95f34195ae4d619d5408438fe4/spec/system/support_interface/staff_spec.rb#L29-L34)), we need to make sure that when a user is invited and we don't have Azure AD enabled that they can still successfully set a password and sign in.